packages/wasm/pkg/README.md

```
@@ -112,9 +119,11 @@ cd ifc-lite
 pnpm install && pnpm dev
 ```
 
-Open <http://localhost:5173> and load an IFC file.
+Open http://localhost:5173 and load an IFC file.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)